conda_forge_tick/migrators/core.py

```diff
@@ -568,10 +568,14 @@ def _test_node_built(node):
 
             return True
 
+        # Special case pinning so we don't need to add it as a dep for every feedstock
```

Comment:
This comment is wrong. I think you mean to say we don't need to add every feedstock to its deps.